lib/mayaUsd/render/vp2RenderDelegate/draw_item.cpp

```diff
@@ -54,4 +51,20 @@ HdVP2DrawItem::~HdVP2DrawItem()
     }
 }
```

Comment:
Do we need to update the dtor to release render items for all geom subsets?

Reply:
yes!